### DISCUSSION TITLE
Does MRTK support Meta Quest Pro?

### DISCUSSION THREAD

**Discussion by ghost** [microsoft/MixedRealityToolkit-Unity#11142] [Q&A] [answered]
I'm planning on buying a meta quest pro and would like to port my existing hololens 2 app. Is MRTK compatible with the new meta quest pro?

https://about.fb.com/news/2022/10/meta-quest-pro-is-now-available/

**Comment by Zee2** [comment, marked as answer]
*Yes!* MRTK3 supports Meta Quest Pro through OpenXR. Meta-specific functionality like passthrough is limited at this time; you may use the OVR plugin/manager to enable those features, but you may see compatibility issues when using the Oculus proprietary plugins alongside MRTK3. We're currently working on making sure you can either use the Oculus-proprietary plugins, or use our OpenXR tooling to enable passthrough on Quest Pro. @keveleigh can elaborate.

For MRTK v2.8, it *should* work through the existing Quest/OpenXR support that exists there, but it is untested as of yet.

**Reply by ghost** [reply]
Awesome!!!!!! MRTK3 rocks!

**Reply by bjornsyse** [reply]
Sounds great. How can one think about eye-tracking. Is that something that would be supported via OpenXR or what's the route there?

**Reply by keveleigh** [reply]
Yep! That should come via OpenXR. There's a cross-vendor interaction profile extension called [XR_EXT_eye_gaze_interaction](https://registry.khronos.org/OpenXR/specs/1.0/html/xrspec.html#XR_EXT_eye_gaze_interaction) (which Oculus/Meta participated in creating) which should be where the eye tracking data is provided.

**Reply by bjornsyse** [reply]
Good news! I get literally zero hits googling for quest pro and that extension name. Anyone with more information on whether that is already supported or any other intel feel free to chirp in!

**Reply by thisisvaze** [reply]
https://github.com/microsoft/MixedRealityToolkit-Unity/discussions/11239#discussioncomment-4217061

**Comment by Florimondable** [comment]
> For MRTK v2.8, it _should_ work through the existing Quest/OpenXR support that exists there, but it is untested as of yet.

Hi,
After three months may I asked again: anyone had tested Meta Quest Pro on MRTK v2.8 ?

**Comment by Sanket2255** [comment]
Mixed Reality Toolkit Foundation Text-To-Speech do not work on Meta Quest when converted project of hololens2 to Meta Quest as @Zee2  said, but other than Text-To-Speech every thing works fine. So any idea everyone?
